app/src/Components/ListTable/ListTableRowButtons.js

```diff
@@ -29,23 +32,34 @@ export const ListTableRowButtons = ({buttons, value}) => {
 }
 
 function _createButton(button, value) {
-  switch(button.function) {
+  switch(button.name) {
     case 'edit':
       return (
           <IconButton key={'edit_' + value} component={Link} to={button.parameter + '/edit/' + value} aria-label="Delete" color="primary">
```

Comment:
Why is the `aria-label` of this button `Delete`?

Reply:
copy-pasta from simple button :(